ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -103,6 +103,8 @@ echo "$(date +%s)" > "${SHARED_DIR}/TEST_TIME_INSTALL_END"
 
 if test "${ret}" -eq 0 ; then
   touch  "${SHARED_DIR}/success"
+  # Save console URL in `console.url` file so that ci-chat-bot could report success
+  echo "https://$(env KUBECONFIG=${dir}/auth/kubeconfig oc -n openshift-console get routes console -o=jsonpath='{.spec.host}')" > "${SHARED_DIR}/console.url"
```

Comment:
Installer [already logs this][1].  Do we need `oc` to fetch it again?  Maybe we can get that structured?  By having the installer add it to `metadata.json`?  Or something?  See the benefits of using the installer's logic [here][2].

[1]: https://github.com/openshift/installer/blob/7ba3f375977b7e2a0adc856db3f258f2c53b8aef/cmd/openshift-install/create.go#L554
[2]: https://github.com/openshift/installer/commit/aafdb628a22ffbe47b0eb7d082e1bf27c909674c

Reply:
We need it in the file so that cluster-bot would [know that the workflow is done](https://github.com/openshift/ci-chat-bot/blob/master/prow.go#L646) and, later on, [show login address](https://github.com/openshift/ci-chat-bot/blob/master/prow.go#L751-L752) without cleaning up the installer output (as we want to redact kubeadmin-password from the output in CI)

Reply:
I like the idea of writing this to `metadata.json` though

Reply:
I don't think that this is appropriate for `metadata.json`.
1. It would be a mutation of the file at the end of the installation rather than added when the file was created.
2. It is not installer-owned information and would not be used at all by the installer, which is the audience for `metadata.json`.

I could get on board with writing it out to a file when the install completes, on par with what is done for the kubeadmin password and the admin kube config.